sbin/autograder/grade_item.py

```diff
@@ -217,8 +218,11 @@ def grade_from_zip(my_autograding_zip_file,my_submission_zip_file,which_untruste
     # unzip autograding and submission folders
     tmp_autograding = os.path.join(tmp,"TMP_AUTOGRADING")
     tmp_submission = os.path.join(tmp,"TMP_SUBMISSION")
-    unzip_this_file(my_autograding_zip_file,tmp_autograding)
-    unzip_this_file(my_submission_zip_file,tmp_submission)
+    try:
+        unzip_this_file(my_autograding_zip_file,tmp_autograding)
+        unzip_this_file(my_submission_zip_file,tmp_submission)
+    except:
+        raise
```

Comment:
Is this an incomplete edit?

Reply:
No, that syntax just means re-raise the exception. I added the try/except more as an acknowledgement that ```unzip_this_file``` is a frequent cause of exceptions than anything. I was logging the exception at first, but it is logged at the next level up, and if the copy fails, the function really shouldn't continue.